scripts/oneoff/categories-baseline.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+"""Usage: categories-baseline.py <g-cloud-version> [--stage=STAGE]
+
+This very basic script is run for our performance analyst. It gives them an idea of what categories services are being
+offered under. It also shows who is a reseller, and who is selling original products. As this script hammers the API to
+get DUNS numbers, I strongly recommend it's only run locally against a recent database dump. So I've written the script
+to default to 'local' and users will need to specifically ask it to run against a different stage.
+
+Options:
+    <g-cloud-version>                                     Which iteration of G-Cloud to target. Use g-cloud-x format
+
+    [--stage=STAGE]                                       Stage to target
+
+    -h, --help                                            Show this screen
+
+"""
+import sys
+
+from dmapiclient.data import DataAPIClient
+
+sys.path.insert(0, '.')
+
+from dmscripts.helpers.auth_helpers import get_auth_token
+from dmutils.env_helpers import get_api_endpoint_from_stage
+from docopt import docopt
+
+if __name__ == "__main__":
+    arguments = docopt(__doc__)
+
+    # Get script arguments
+    stage = arguments['--stage'] or 'local'
+    version = arguments['<g-cloud-version>']
+
+    # Set defaults, instantiate clients
+    data_api_client = DataAPIClient(
+        base_url=get_api_endpoint_from_stage(stage),
+        auth_token=get_auth_token('api', stage)
+    )
+
+    hosting_services = []
+    software_services = []
+    support_services = []
+    for service in data_api_client.find_services_iter(framework=f'{version}', status='published'):
+        if service.get('lot') == 'cloud-support':
+            support_services.append(service)
+        elif service.get('lot') == 'cloud-hosting':
+            hosting_services.append(service)
+        elif service.get('lot') == 'cloud-software':
+            software_services.append(service)
+
+    lots = {
+        'hosting': hosting_services,
+        'software': software_services,
+        'support': support_services
+    }
+    for key, value in lots.items():
+        with open(f'{key}-categories-{version}.tsv', 'w') as f:
```

Comment:
I think one file per lot is a good idea, but can we 1) mention that this is the expected output in the top docstring 2) maybe offer an optional destination folder arg, so the files don't have to hang around in `scripts/oneoff`?